Tests/store_packs.py

```diff
@@ -236,6 +239,17 @@ def zip_pack(self):
                     dirs[:] = [d for d in dirs if d not in Pack.EXCLUDE_DIRECTORIES]
 
                     for f in files:
+                        # skipping unzipping of unwanted files
+                        if f.startswith('.') or f in [Pack.AUTHOR_IMAGE_NAME, Pack.USER_METADATA]:
+                            print_warning(f"Skipping zipping {f} for {self._pack_name} pack")
+                            continue
+
+                        current_directory = root.split(os.path.sep)[-1]
+
+                        if current_directory == 'Misc' and not fnmatch.fnmatch(f, 'reputation-*.json'):
+                            # reputation in old format aren't supported in 6.0.0 server version
+                            print_warning(f"Skipped zipping {f} for {self._pack_name} pack")
```

Comment:
@IkaDemisto soul dent you add `continue` here?

Reply:
@yuvalbenshalom correct